### PR TITLE
Nifi 1958

### PR DESCRIPTION
**Pull request by Mistasong39**
Added Functionality to convert weeks to an existing `TimeUnit`. Did not add weeks as a destination `TimeUnit`.

### CODE REVIEW

**Comment by alopresto** [comment]
Reviewing...

File: nifi-commons/nifi-utils/src/main/java/org/apache/nifi/util/FormatUtils.java [hunk 1]
```diff
@@ -41,8 +41,9 @@
     private static final String MINS = join(UNION, "m", "min", "mins", "minute", "minutes");
     private static final String HOURS = join(UNION, "h", "hr", "hrs", "hour", "hours");
     private static final String DAYS = join(UNION, "d", "day", "days");
+    private static final String WEEKS = join(UNION, "w", "wk", "week", "weeks");
```
Comment (alopresto) [inline nifi-commons/nifi-utils/src/main/java/org/apache/nifi/util/FormatUtils.java]: What about `wks`?

File: nifi-commons/nifi-utils/src/main/java/org/apache/nifi/util/FormatUtils.java [hunk 2]
```diff
@@ -176,6 +177,12 @@ public static long getTimeDuration(final String value, final TimeUnit desiredUni
             case "days":
                 specifiedTimeUnit = TimeUnit.DAYS;
                 break;
+            case "w":
+            case "wk":
```
Comment (alopresto) [inline nifi-commons/nifi-utils/src/main/java/org/apache/nifi/util/FormatUtils.java]: If `wks` is added in the list above, it should be added as a case here.

File: nifi-commons/nifi-utils/src/main/java/org/apache/nifi/util/FormatUtils.java (hunk 1 above)
Comment (Mistasong39) [inline nifi-commons/nifi-utils/src/main/java/org/apache/nifi/util/FormatUtils.java]: I will add that

**Comment by alopresto** [comment]
Thanks for adding `wks`. I built locally and verified the checkstyle. Please rebase and squash all the commits and push, and I will merge into `master` and `0.x`.

File: nifi-commons/nifi-utils/src/main/java/org/apache/nifi/util/FormatUtils.java (hunk 2 above)
Comment (Mistasong39) [inline nifi-commons/nifi-utils/src/main/java/org/apache/nifi/util/FormatUtils.java]: I will add that.

**Pull request closed without merging**